.github/workflows/test.yml

```
@@ -24,6 +24,9 @@ jobs:
       - name: Lint
         run: npm run lint
 
+      - name: Npm Build
```

style: just for consistency
```suggestion
      - name: Build
```